.github/workflows/test.yml

```
@@ -208,3 +213,21 @@ jobs:
       - name: Run the tests
         working-directory: ./myadmin
         run: npm run test
+
+  update-sandbox-repository:
+    runs-on: ubuntu-latest
+    # Only run on new tags that target a release (not latest nor next) and avoid alpha and beta tags
+    if: github.event_name == 'push' && github.ref_type == 'tag' && github.ref_name == 'refs/tags/v*' && !contains('beta', github.ref_name) && !contains('alpha', github.ref_name)
+    needs: [typecheck, simple-example-typecheck, unit-test, e2e-test]
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v3
+        - name: Use Node.js LTS
+          uses: actions/setup-node@v3
+          with:
+              node-version: '16.x'
```

```suggestion
              node-version: '18.x'
```

To be consistent with other jobs of the workflow